clang/test/CodeGenSYCL/save-user-names.cpp

```diff
@@ -0,0 +1,74 @@
+// RUN:  %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+
+// Test to verify that user specified names are retained in openCL
+// kernel argument names.
+
+#include "Inputs/sycl.hpp"
```

Comment:
I think common tests writing style says that we should do `#include "sycl.hpp"` and `internal-isystem %S/Inputs` . This is not important for this particular test, but it is important when a test is about warning messages. People tend to copy-paste tests, so it is better to keep them all in the same style.

Reply:
Done!